src/components/dashboard/dashboard.tmpl.html

```diff
@@ -110,8 +110,8 @@ <h3 translate>impac.dashboard.it_looks_like</h3>
           <div class="row top-buffer-4">
             <div class="col-md-8 col-md-offset-2">
               <div class="info-text">
-                <p><b>{{dhbErrorsConfig.firstTimeCreated.first}}</b></p>
-                <p>{{dhbErrorsConfig.firstTimeCreated.second}}</p>
+                <p><b translate>{{dhbErrorsConfig.firstTimeCreated.first}}</b></p>
```

Comment:
If 'dhbErrorsConfig.firstTimeCreated.first' returns a string that does not match a translation variable name in dictionary, translation function will return the string which will be equal variable name.

For example:
<p translate>{{'name of a non-existent variable'}}</p>
will be:
<p>name of a non-existent variable</p>